tests/models/test_watchlists.py

```diff
@@ -0,0 +1,22 @@
+from tastyworks.models import watchlists
```

Comment:
Same comment as the one above for `test_symbol_search.py`

Reply:
I think this one differs a little since watchlists are a model? You may have to help me understand if I'm not following here.

Reply:
Reason I'm suggesting is that the `models` module should hold abstractions & representations of the different data types we could get from tastyworks. How they are applied & used should either be up to the customer (if we're an API/SDK, that's usually the case 90% of the time) or at least in a separate folder denoting their common purpose.

In software engineering, on average it's a good idea to separate business-logic from use-cases :)

Reply:
I'm understanding the idea of separating these two functions, but I'll admit there's some confusion between what I would consider business-logic and what I would consider use-case.

I feel like `WatchlistGroup` and `Watchlist` are both models we build from calls to the API and return to the consumer.  So, would it make sense to keep this in the models module?

Reply:
> I feel like WatchlistGroup and Watchlist are both models we build from calls to the API and return to the consumer. So, would it make sense to keep this in the models module?

Completely agreed :) Having those models remain here makes perfect sense. I just think that a file that's used for testing them in local development doesn't :)

Reply:
I took a dive off the cliff yesterday into `pytest` and `unittest` and this completely makes sense to me now.  I see the validity of not including this test as part of the package.